### ISSUE TITLE
training performance

### ISSUE THREAD

**Issue by pillar02** [kuangliu/pytorch-ssd#1]
hey, thanks for you great code. it's clean and easy to follow. 
But have you tested the training performance?
I tried to train on my own data. It seems to overfit on the training data (I also trained this data with mxnet ssd code, and that seems to give better performance).
I noticed the L2Norm module is different from the original paper (it is fixed to 20). And I changed it to alow adjusting.
I also added some extra data augmentation into the training process.
Any idea of reasons?

**Comment by kuangliu** [comment]
This is some experiment code I wrote for learning, not aiming for accuracy.
If you do use pretrained VGG model, to my best knowledge the reason might be:
  1. potential bug in my code (highly possible).
  2. more data augmentation is needed.

You can tweak the code, if you find something helpful to the performance, you are very welcome to make a PR. And I'll look into the code when I get time.
